src/framework/components/gsplat/component.js

```diff
@@ -206,6 +212,31 @@ class GSplatComponent extends Component {
         return this._instance?.material ?? this._materialTmp ?? null;
     }
 
+    /**
+     * Sets whether the full spherical-harmonic solver is enabled when rendering SOGS data.
```

Comment:
I'd prefer additional info of what this means for the user. Mention quality vs performance, for sogs specifically.

Reply:
`fullSH` means more to me than `hqSH` (and fullSH != enableSH), but if you both agree on that naming I'm happy to change it.

Reply:
I'd probably go with `highQualitySH`. But explain this applies to SOGS only.

Reply:
I'll take that.

Reply:
Yeah, I prefer that.

Reply:
you could say the same thing about `approxSH`. the name doesn't imply it's fast, so why would anyone use it?

Reply:
yeah
For DOF, I have `highQuality` setting as it seemed the best.

Reply:
(BTW for me it's an interesting exercise to try tease out what the best name could be in this case, but I'm not actually fussed either way. If you guys agree on a name I will make it so).